### DISCUSSION TITLE
S3 Gateway

### DISCUSSION THREAD

**Discussion by wibed** [soto-project/soto#647] [General]
like in the following issue i would like to run vapor against a s3 gateway to
https://github.com/soto-project/soto/issues/462

sadly, minio deprecates its gateway https://github.com/minio/minio/issues/14331

now i am looking for an alternative and the only option that has sprung up is
https://github.com/gaul/s3proxy
https://github.com/scality/cloudserver

s3proxy is very limited in its feature set.
zenko cloudserver runs on node and its primary goal not to be the gateway itself
does any one of you have an alternative for the minio gateway?

**Comment by adam-fowler** [comment]
How do you intend to use the S3 Gateway?

**Comment by wibed** [comment]
ideally i would like to scale over multiple cloudservices.

for example using local storage for hot data and some aws stuff for some colder ones.

**Reply by adam-fowler** [reply]
ok then I don't know of any other services. If you were just using it for local mocking there are plenty of alternatives.

**Reply by wibed** [reply]
if you dont know, then i presume my chances fairly low.
thanks anyways 💯
